packages/kit/src/ignore.ts

```
@@ -0,0 +1,27 @@
+import { existsSync, readFileSync } from 'fs'
+import ignore from 'ignore'
+import { join, relative } from 'pathe'
+import { useNuxt } from './context'
+
+/**
+ * Return a filter function to filter an array of paths
+ */
+export function isIgnored (pathname: string): boolean {
+  const nuxt = useNuxt()
+
+  if (!nuxt._ignore) {
```

Maybe we could improve it later, `useNuxt()` silently returns `null` when context is not available. This causes this line fail with strange error like cannot find prop `_ignore` of undefined. We could maybe make an error by default for `useNuxt` or allow an optional argument to throw with a custom message